packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2.ts

```diff
@@ -17,6 +17,27 @@ export interface SecondaryAddressProps {
   readonly cidrBlockName: string;
 }
 
+/**
+ * Additional props needed for BYOIP IPv6 address props
+ */
+export interface Ipv6PoolSecondaryAddressProps extends SecondaryAddressProps {
+  /**
+   * ID of the IPv6 address pool from which to allocate the IPv6 CIDR block
```

Comment:
nit: end public docstrings with a period `.`
```suggestion
   * ID of the IPv6 address pool from which to allocate the IPv6 CIDR block.
```